src/main/kotlin/com/mutualmobile/praxisspringboot/entities/user/DBUserProject.kt

```diff
@@ -0,0 +1,12 @@
+package com.mutualmobile.praxisspringboot.entities.user
+
+import com.mutualmobile.praxisspringboot.entities.BaseEntity
+import javax.persistence.Entity
+import javax.persistence.Table
+
+@Entity
+@Table(name = "user_project")
```

Comment:
@shubhamsinghmutualmobile can we call this user_project_assignment

Reply:
Just the table? Or the class as well (i.e. DBUserProjectAssignment instead of DBUserProject)?

Reply:
both